study-group/README.md

```diff
@@ -0,0 +1,130 @@
+# Toast Study Group
+
+This document compares the capabilities of various popular toast libraries.
+
+## Background
+A toast is a UI element that provides valuable feedback to a user about the actions they perform on the web or their mobile device.
```

Comment:
"web or mobile device" is a bit of a strange dichotomy. Maybe just "while using an app" or similar, with no platform discussions.

Reply:
```suggestion
A toast is a UI element that provides valuable feedback to a user about the actions they perform while using an app.
```